plugins/se/se.cpp

```diff
@@ -195,10 +195,12 @@ static bool tryInit(const std::multimap< std::wstring, unsigned long long int >
 	};
 
 	for (const auto &name : names) {
-		const auto id = ProcessBase::find(name, pids);
-		if (!id) {
-			continue;
+		const auto iter = pids.find(std::wstring(name.cbegin(), name.cend()));
+		if (iter == pids.cend()) {
+			return false;
```

Comment:
It seems to me that the code logic was modified here. Wouldn't we want to ``continue`` instead of ``return false;``?

Reply:
Yup, I accidentally copied the code from `cod2.cpp`.